android/app/src/main/java/com/zulipmobile/MainActivity.java

```diff
@@ -1,8 +1,6 @@
 package com.zulipmobile;
 
 import com.facebook.react.ReactActivity;
-import com.smixx.fabric.FabricPackage;
-import com.oblador.vectoricons.VectorIconsPackage;
```

Comment:
There is only import no usage. https://github.com/zulip/zulip-mobile/blob/master/android/app/src/main/java/com/zulipmobile/MainActivity.java (so may be automatically removed by Android Studio)